src/libstd/error.rs

```diff
@@ -552,6 +552,13 @@ impl Error for char::ParseCharError {
     }
 }
 
+#[unstable(feature = "try_reserve", reason = "new API", issue = "48043")]
+impl Error for alloc::collections::TryReserveError {
+    fn description(&self) -> &str {
```

Comment:
The description method is deprecated - it should be omitted here.

Reply:
Ok, I removed it.  

It was only soft-deprecated in stable, so I had added it just in case.